src/apps/create-patron-user-info/helper.tsx

```diff
@@ -0,0 +1,17 @@
+import { UseTextFunction } from "../../core/utils/text";
```

Comment:
Our convention is to call helper files that do not have jsx rendering: `helper.ts`